bundles/org.openhab.binding.tplinkrouter/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please generate the i18n properties file (mvn i18n:generate-default-translations).